lib/constants/index.js

```diff
@@ -11,6 +11,13 @@
  * of collisions with other plugins
  */
 const PLUGIN_NAMESPACE = 'bwallet';
+// prefix for reducers
+const PLUGINS = 'plugins';
+const REDUCE_INTERFACE_NAMESPACE = '@bpanel/bwallet/interface';
+const REDUCE_NODE_NAMESPACE = '@bpanel/bwallet/node';
+const REDUCE_CHAIN_NAMESPACE = '@bpanel/bwallet/chain';
+const REDUCE_WALLETS_NAMESPACE = '@bpanel/bwallet/wallets';
```

Comment:
Should the plugin namespace just be used as a constant with template literals here? 

Also wondering about the decision to have each of these top level rather than basically as child stores under the namespace? I could see this ending up making the plugins store much more cluttered

<img width="769" alt="screen shot 2018-09-12 at 11 35 05 am" src="https://user-images.githubusercontent.com/4344978/45446691-661d4c80-b682-11e8-8e8c-fc6fd37b2125.png">

Reply:
nested reducers and/or combineReducers: 
https://github.com/reduxjs/redux/issues/738
http://www.ematipico.com/javascript/redux/nested-reducers/

Reply:
It's a little obfuscated but this is how we actually manage the plugin reducers in bpanel to enable for the namespaced reducers nested in the `plugins` reducer: https://github.com/bpanel-org/bpanel/blob/development/webapp/plugins/plugins.js#L263

Maybe this would be helpful to add to the [reducers documentation](https://bpanel.org/docs/api-reducers.html#pluginreducers)... 🤔